test/ingress/aws/aws_test.go

```diff
@@ -18,129 +18,37 @@
 package aws
```

Comment:
A lot of happening here in this commit, can we break it down e.g. to bug fix and refactor commits, so it's more clear what has been fixed? Otherwise I only see code moved and it's not obvious from where new values in struct come from.